DOC-2325: Add new `sandbox_iframes_exclusions` option to `content-filtering.adoc` to support `sandbox_iframes`. (#3155)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/sandbox_iframes_exclusions.adoc

[[sandbox-iframes-exclusions]]
== `sandbox_iframes_exclusions`

> This option holds a list of URL host names to be excluded from iframe sandboxing when sandbox_iframes is set to true.
> 
> [NOTE]
> `sandbox_iframes` option is required to be set in the editor configuration for this option to affect the  is not enabled, this option will have no effect.

Reviewer: ```suggestion
This option allows you to specify a list of domains to exclude from having the `sandbox=""` attribute applied, when the `sandbox_iframes` option is enabled. This option takes an array of strings, each denoting at least the top and second level of a domain to be excluded. By default this option is set to an array of domains which are provided in embed code by popular websites. To enable [sandboxing](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox) on all iframes, set this option to an empty array `[]`.

[NOTE]
When the `sandbox_iframes` option is disabled, all domains will be excluded from sandboxing.
```

Author: thank you for updating this, i wasn't sure how to put this one together.

> *Type:* `+String+`, `+Array+`
> 
> *Possible values:* `true`, `false`
> 
> *Default value:* `true`

Reviewer: ```suggestion
*Type:* `+Array+`

*Default value:*
[source,js]
----
[
  'youtube.com',
  'youtu.be',
  'vimeo.com',
  'player.vimeo.com',
  'dailymotion.com',
  'embed.music.apple.com',
  'open.spotify.com',
  'giphy.com',
  'dai.ly',
  'codepen.io'
]
----
```

[[sandbox-iframes-exclusions]]
== `sandbox_iframes_exclusions`

> This option allows you to specify a list of domains to exclude from having the `sandbox=""` attribute applied, when the `sandbox_iframes` option is enabled. This option takes an array of strings, each denoting at least the top and second level of a domain to be excluded. By default this option is set to an array of domains which are provided in embed code by popular websites. To enable [sandboxing](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox) on all iframes, set this option to an empty array `[]`.

Reviewer 2: ```suggestion
This option allows a list of domains to be specified that should be excluded from having the `sandbox=""` attribute applied when the `sandbox_iframes` option is enabled. This option takes an array of strings, each denoting at least the top and second level of a domain to be excluded. By default, this option is set to an array of domains that are provided in embed code by popular websites. To enable [sandboxing](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox) on all iframes, set this option to an empty array `[]`.
```

[NOTE]

> When the `sandbox_iframes` option is disabled, all domains will be excluded from sandboxing.

Reviewer 2: ```suggestion
When the `sandbox_iframes` option is set to `false`, all domains will be excluded from sandboxing.
```

*Type:* `+Array+`

*Default value:*
[source,js]
----
[
  'youtube.com',
  'youtu.be',
  'vimeo.com',
  'player.vimeo.com',
  'dailymotion.com',
  'embed.music.apple.com',
  'open.spotify.com',
  'giphy.com',
  'dai.ly',
  'codepen.io'
]
----

=== Example: using `sandbox_iframes_exclusions` option

[source,js]
----
tinymce.init({
  selector: "textarea",

>   sandbox_iframes: false,

Reviewer 2: The `sandbox_iframes_exclusions` would not do anything otherwise
```suggestion
  sandbox_iframes: true,
```
